study.md

```diff
@@ -30,11 +30,14 @@ is listed as an **Included Module** for Array.
 
 List at least two instance method names (other than `count`) that Array and
 Enumerable share. An included module provides behavior to instances of the
-including class. Why might Array redefine methods included from Enumerable?
+including class. Why might Arrays redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+:any? will got through the whole array and see if 'any' of what you are looking for exists.
+:find will find anything you are looking for if it exists.
+
+Whatever program we are using doesn't know about enumerables therefore we have to specify with the included module with arrays. Otherwise it would treat it like any other array.
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.